src/status_im/ui/components/chat_icon/screen.cljs

```diff
@@ -10,19 +10,24 @@
 
 ;;TODO REWORK THIS NAMESPACE
 
+(def get-name-first-char
+  (memoize
+   (fn [name]
+     ;; TODO: for now we check if the first letter is a #
+     ;; which means it is most likely a public chat and
+     ;; use the second letter if that is the case
+     ;; a broader refactoring should clean up upstream params
+     ;; for default-chat-icon
+     (string/capitalize (if (and (= "#" (first name))
```

Comment:
How about `(re-find #"[a-z]" (str/lower-case name))`. This will return the first alphabet that appears in the string.

```clojure
(re-find #"[a-z]" "#$$$Status") ; => S

Reply:
this is really old code, don't want to touch it in this PR